docker-compose.yml

```diff
@@ -11,6 +11,16 @@ services:
       POSTGRES_DB: "ctdb"
     volumes:
       - "postgres_data:/var/lib/postgresql/data"
+  test_db:
+    container_name: test_ctdb_postgres
+    image: postgres:latest
+    environment:
+      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
+      POSTGRES_DB: "ctdb"
+    ports:
+      - "5433:5432"
```

Comment:
Any particular reason we expose 5433 instead of 5432? I don't think it's a problem but I am curious.

Reply:
I want to say that this was so that the dev and test databases wouldn't be exposing the same port? Though I don't even know if this is how ports work so could be totally off base. Will make a note in the modified commit msgs